Tallies/tallyCodes.f90

```diff
@@ -22,6 +22,7 @@ module tallyCodes
   integer(shortInt),parameter,public :: abs_FATE  = 5000 ,&
                                         leak_FATE = 5001 ,&
                                         lost_FATE = 5002 ,&
-                                        aged_FATE = 5003
+                                        aged_FATE = 5003 ,&
+                                        time_FATE = 5004
```

Comment:
aged_FATE is time_FATE - as in a neutron has aged too much. time_FATE can be removed and replace in the relevant places by aged.